public/app/types/panel.ts

```diff
@@ -0,0 +1,7 @@
+import TableModel from 'app/core/table_model';
+
+export interface TableTransform {
```

Comment:
this is a type specific to the table panel, can you move it the table panel folder?